core/src/main/scala/cats/instances/sortedSet.scala

```diff
@@ -90,43 +91,27 @@ trait SortedSetInstancesBinCompat0 {
   }
 }
 
-class SortedSetOrder[A: Order] extends Order[SortedSet[A]] {
-  def compare(a1: SortedSet[A], a2: SortedSet[A]): Int =
-    Order[Int].compare(a1.size, a2.size) match {
-      case 0 => StaticMethods.iteratorCompare(a1.iterator, a2.iterator)
-      case x => x
-    }
-
-  override def eqv(s1: SortedSet[A], s2: SortedSet[A]): Boolean =
-    StaticMethods.iteratorEq(s1.iterator, s2.iterator)
+private[instances] trait SortedSetInstancesBinCompat1 extends LowPrioritySortedSetInstancesBinCompat1 {
```

Comment:
same concern here, introducing a bincomp trait to fix an inconsistent placement while we should be able to fix this right after 2.0 release without it